### DISCUSSION TITLE
No phonemizer found for language es

### DISCUSSION THREAD

**Discussion by seonake** [coqui-ai/TTS#1465] [General Q&A] [answered]
Hi,
I guess this must be preaty easy... but i dont get it right.
From pre-trained model ("tts_models/es/mai/tacotron2-DDC"), I get this error:

---->               ValueError: No phonemizer found for language es.
                        You may need to install a third party library for this language.

Where can I get this library?
Many thanks

**Comment by seonake** [comment]
Nobody helps?
Please, i kindly request ideas... Many thanks

**Comment by erogol** [comment]
"You may need to install a third party library for this language."

Install espeak

**Reply by seonake** [reply]
Hi, many thanks... but I think I did.
pip install speake3
https://sourceforge.net/projects/espeak/files/latest/download

I am using Ananconda, W10.

**Reply by seonake** [reply]
In fact I can use Espeak by itself... :(

**Comment by seonake** [comment, marked as answer]
I fixed it! 
If someone else has this problem, add espeak/command-line to path (enviroment varibale).
Thanks
